### PR TITLE
Close the database connection

### PR DESCRIPTION
**Pull request by sbrunner**
Fix [GEO-6793](https://jira.camptocamp.com/browse/GEO-6793)

### CODE REVIEW

**Comment by julianhafner** [comment]
works for me :+1:

File: core/src/main/java/org/mapfish/print/output/AbstractJasperReportOutputFormat.java [hunk 1]
```diff
@@ -194,17 +194,22 @@ public final Print getJasperPrint(
...
+          connection =
+              DriverManager.getConnection(
+                  template.getJdbcUrl(), template.getJdbcUser(), template.getJdbcPassword());
+        } else {
+          connection = DriverManager.getConnection(template.getJdbcUrl());
+        }
 
+        print = fillManager.fill(jasperTemplateBuild.getAbsolutePath(), values.asMap(), connection);
+      } finally {
+        if (connection != null) {
```
Comment (sebr72) [inline core/src/main/java/org/mapfish/print/output/AbstractJasperReportOutputFormat.java:209]: To be safe better  add: "&& !connection.isClosed()"

**Review by sebr72 — CHANGES_REQUESTED** [review]
As discussed

**Review by sebr72 — APPROVED** [review]
With changes as discussed

File: core/src/main/java/org/mapfish/print/output/AbstractJasperReportOutputFormat.java (hunk 1 above)
Comment (jwkaltz) [reply core/src/main/java/org/mapfish/print/output/AbstractJasperReportOutputFormat.java]: Resolving conversation, as proposed change has been committed

**Comment by jwkaltz** [comment]
@sebr72 @sbrunner do you what the cause of the CI error in the step "Auto reviews updates / Auto reviews updates (pull_request)" is?
Is it relevant, or can we merge?

**Pull request merged**